### ISSUE TITLE
not being able to get device integrity to get access to using google pay

### ISSUE THREAD

**Issue by defalt2003** [chiteroman/PlayIntegrityFix#211]
Hello I have been having issues again with trying to get the Integrity Fix on 15.1 working again after resetting my phone due to a boot loop and I have reinstalled magisk, then I have also installed the Integrity Fix and its BASIC but I can't add my card so I am unable to use google wallet/ pay

**Comment by Thisisauser6443** [comment]
Google banned the fingerprint, which was used in this module, so you'll need to find your own. There's a decent number of guides and resources lying about, on XDA/GitHub, so you might wanna have a look at those, but at the end of the day, it'll usually come down to trial and error

**Comment by defalt2003** [comment]
Ahh ok thank you for letting me know as I wasn't aware 🙂
________________________________

**Comment by Brons0n** [comment]
Will it be updated soon?

**Comment by Hope-Upstairs** [comment]
Replaced the fingerprint, passed the checks, but it still gives me an error.

![IMG_20240116_155915](https://github.com/chiteroman/PlayIntegrityFix/assets/58907455/b6b9e696-727f-4cff-84e3-dcc847aea739)
![IMG_20240116_155836](https://github.com/chiteroman/PlayIntegrityFix/assets/58907455/dc2f2308-f99b-4034-b82a-87179487af07)

![Screenshot_2024-01-16-16-00-17-060_com google android gms](https://github.com/chiteroman/PlayIntegrityFix/assets/58907455/59398866-f8cc-4918-ad73-c7e9e6d731a6)

**Comment by Hope-Upstairs** [comment]
My bad, just got a pizza. it went though. The app still shows the error for some reason, though.

**Comment by lmn451** [comment]
Hey all,
Probably should be in new issue, but lets check if we can resolve it here?

Pixel 7a | UQ1A.240105.004, Jan 2024 |  Magisk 26.4 | play integrity fix 15.1

Today out of nowhere got this 

<img width="434" alt="image" src="https://github.com/chiteroman/PlayIntegrityFix/assets/14910239/ca867140-10de-44ee-b55c-58a570a00d76">

![IMAGE 2024-01-16 22:25:37](https://github.com/chiteroman/PlayIntegrityFix/assets/14910239/a07e55ee-3e78-43ed-a8c8-50a886649bb4)
Will retest it tomorrow. Any suggestions? 

Basically I use Magisk for call recordings, so is there simplier way to do this?

**Comment by DKNight-9207** [comment]
I have same questions, too.

I tested it yesterday, but i only pass basic
Is there anything I need to change?
![Screenshot_2024-01-17-12-38-50-764_ly.pcufzxm.vdiiebow.jpg](https://github.com/chiteroman/PlayIntegrityFix/assets/68389902/3b6a11c2-e52d-489a-9d7f-df9a319b7979)

**Comment by XKangs** [comment]
same here. I am using the 15.1 with my own fingerprint. it was fine  until today, I think Google makes some changes on their end

**Comment by Thisisauser6443** [comment]
Everything's fine on my end, so I'd assume it's just Google banning more fingerprints, again 

![Screenshot_20240117-135225~2.png](https://github.com/chiteroman/PlayIntegrityFix/assets/75092921/4a49abdf-216c-4be0-a719-cf47aaae2410)

**Comment by aveao** [comment]
I get the visual bug in gpay too, but with a custom signature and 15.2, after a restart my gpay payments go through.
